knora/dsplib/schemas/data.xsd

```diff
@@ -65,7 +65,7 @@
         <xs:simpleContent>
             <xs:extension base="knoracolor_type">
                 <xs:attribute name="comment" type="xs:string"/>
-                <xs:attribute name="permissions" type="xs:NCName" use="required"/>
+                <xs:attribute name="permissions" type="xs:NCName" use="optional"/>
```

Comment:
It is in fact possible to pass no permissions at all. No one has then access to the property except from the sysadmin.

Reply:
makes sense that this is technically possible? But does it also make sense to allow this in terms of usage? Do we want to have this in our system?